### PR TITLE
Use `detect` over `select...first`

### PR DESCRIPTION
**Pull request by minuscorp**

### CODE REVIEW

File: services/services.rb [hunk 1]
```diff
@@ -77,6 +77,8 @@ def self.configuration_git_repo
     end
 
     def self.ci_user
+      require "pry"
+      binding.pry
```
Comment (snatchev) [inline services/services.rb]: Looks like some debugging code got left in here :)

File: features/project/views/new_project.erb [hunk 2]
```diff
@@ -19,7 +19,7 @@
         <% repos.each do |repo| %>
           <tr style="cursor: pointer;">
             <td>💚</td>
-            <td onclick="document.location = '/projects_erb/add/<%= repo[:name] %>';"><%= repo[:name] %></td>
+            <td onclick="document.location = '/projects_erb/add/<%= CGI.escape(repo.full_name) %>';"><%= repo.full_name %></td>
```
Comment (snatchev) [inline features/project/views/new_project.erb]: is there a way to have this applied to all urls? I have a hunch we're constantly going to be overlooking this.

File: features/project/project_controller.rb [hunk 3]
```diff
@@ -70,11 +70,32 @@ class ProjectController < AuthenticatedControllerBase
       erb(:new_project, locals: locals, layout: FastlaneCI.default_layout)
     end
 
+    get "#{HOME}/lanes/*/*/*" do |org, repo_name, branch|
```
Comment (snatchev) [inline features/project/project_controller.rb]: Could you explain the `/*/*/*` part of this route?

**Comment by minuscorp** [comment]
The git tree went crazy...

**Pull request closed without merging**